ci/test/pipeline.template.yml

```diff
@@ -501,7 +501,7 @@ steps:
         plugins:
           - ./ci/plugins/mzcompose:
               composition: platform-checks
-              args: [--scenario=RestartSourcePostgres, --check=PgCdc]
+              args: [--scenario=RestartSourcePostgres, --check=PgCdc, --check=PgCdcNoWait, --check=PgCdcMzNow, --check=SshPg, --check=DebeziumPostgres]
```

Comment:
Do we need to increase the timeout if we add more checks?

Reply:
No, ran in 3 min, timeout is 30 min.